oozie/src/main/java/org/apache/falcon/oozie/OozieBundleBuilder.java

```diff
@@ -106,11 +108,24 @@ public Properties build(Cluster cluster, Path buildPath) throws FalconException
         properties.setProperty(AbstractWorkflowEngine.NAME_NODE, ClusterHelper.getStorageUrl(cluster));
 
         //Add libpath
-        String libPath = getLibPath(buildPath);
-        if (libPath != null) {
-            properties.put(OozieClient.LIBPATH, getStoragePath(libPath));
+        String path = getLibPath(buildPath);
```

Comment:
Adding storage LibPath is inside the function Properties build(). I think String storageLibPath should be returned from function and then add the returned string as value to OozieClient.LIBPATH.